Insider - Cloud Mode Actions Documentation (#4862)
segmentio/segment-docs: src/connections/destinations/catalog/insider-cloud-mode-actions/index.md

## Insider Cloud Mode (Actions) Destination

{% include content/plan-grid.md name="actions" %}

> [Insider](https://useinsider.com/integration/segment/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) Growth Management Platform (GMP) helps digital marketers drive growth across the funnel. Insider GMP helps marketers deliver personalized journeys across the web, mobile web, mobile apps, messaging, email, and ad channels using the unified data.

Reviewer: This looks like the same intro as the non-Actions Insider Destination: is that intentional?

> [Insider](https://useinsider.com/integration/segment/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) Growth Management Platform (GMP) helps digital marketers drive growth across the funnel. Insider GMP helps marketers deliver personalized journeys across the web, mobile web, mobile apps, messaging, email, and ad channels using the unified data.

Reviewer 2: Intro refers to the what insider does, as we are the same company for both of the destinations, that shouldn't be a problem I believe @forstisabella

This destination is maintained by Insider. For any issues with the destination, contact the [Insider Support team.](mailto:insiderhelp@useinsider.com)

> success "" Good to know: This page is about the [Actions-framework](https://segment.com/docs/connections/destinations/actions/) Insider Segment destination. There's also a page about the [non-Actions Insider destination](https://segment.com/docs/connections/destinations/catalog/insider/). Both of these destinations receives data from Segment.

Reviewer: I added some frontmatter to the page that will auto-populate this information in a box at the top of the page, so you can go ahead and remove it in this part of the page!
```suggestion
```

{% include content/ajs-upgrade.md %}

## Benefits of Insider (Actions) vs Insider Classic

Insider (Actions) provides the following benefits over the classic Insider destination:

- **Adjustable Mappings**: By using Insider (Actions), you can easily adjust mapping of your events and attributes and map them to Insider events and attributes.
- **Data Consistency: Pre-Built Mappings can help you to integrate Insider faster and ensure data consistency between Insider and Segment platform.**

## Getting started

1. From the Segment web app, click Catalog, then click Destinations.

2. Find the Destinations Actions item in the left navigation, and click it.

3. Click Configure Insider.

> 4. Select an existing Source to connect to Insider(Actions).

Reviewer: ```suggestion
4. Select an existing Source to connect to Insider Cloud Mode (Actions) and click **Next**.

5. Give your destination a name and click **Create destination*.
```

> 5. Add the following settings to your Insider Destinations

Reviewer: ```suggestion
5. On the settings page for your destination, add the following settings:
```

>    1. **Account Name:** Your Insider Account (Partner) Name.

Reviewer: ```suggestion
   - **Account Name**: Your Insider Account (Partner) Name.
```

>    2. **API Key: Your Unified Customer Database API Key, see how you can generate API key from [here](https://academy.useinsider.com/docs/api-authentication-tokens#generate-api-key).**

Reviewer: ```suggestion
   - **API Key**: Your Unified Customer Database API Key, see how you can generate API key in the [Insider Academy API Authentication Tokens](https://academy.useinsider.com/docs/api-authentication-tokens#generate-api-key){:target="_blank”} documentation.**
 6. Enable your destination and click **Save**.
```

> (delete after reading) The line below renders a table of connection settings (if applicable), Pre-built Mappings, and available actions.

Reviewer 3: ```suggestion
```

Reviewer 2: @emreisik95 Let's delete that, I forgot I believe :D

{% include components/actions-fields.html %}

(delete after reading) Additional Context

> Include additional information that you think will be useful to the user here. For information that is specific to an individual mapping, please add that as a comment so that the Segment docs team can include it in the auto-generated content for that mapping.

Reviewer 3: You might consider adding a comment about the types of identifiers Insider accepts. 
You might also mention which Mappings are enabled by default when the Integration created - and which Mappings need to be enabled by the customer?

Reviewer 2: I'm not sure, I though those would be added by default. No? @joe-ayoub-segment

Reviewer 3: Hi @parkaninsider , @emreisik95 , 

There are multiple different 'identifiers' to consider with this Integration. 
uuid, segment_anonymous_id, and then there's also potentially email, which is controlled by the email_as_identifier field. 

I think it would be a good idea to explain the above in a little more detail to the user in a dedicated paragraph. Would that be possible?

Reviewer 2: Yes let me do that, thanks @joe-ayoub-segment 

@emreisik95 can you add the following line and replace the place Joe highlighted please.

**IDENTITY**

The default behavior is for Segment to pass the userID associated with the user as the uuid (unique user ID). You can easily change the identity attribute to email or phone number by utilizing the Treat Email/Phone Number as Identifier so that users can be matched by using UUID, email and phone number. Insider also receives segment_anonymous_id for every request to match user data between platforms which enables sending anonymous users as well.

Reviewer: There is one component that's autogenerated. It's the destination box at the top of the page, like in the following screenshot:
![destination info box for the Insider Cloud Mode (Actions) destination](https://github.com/segmentio/segment-docs/assets/92472883/c4d726de-e733-44b2-9aa4-5de04f18a00b)
